pkgs/tools/package-management/dnf5/default.nix

```
@@ -0,0 +1,95 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, createrepo_c
+, gettext
+, help2man
+, pkg-config
+, cppunit
+, fmt
+, glib
+, json_c
+, libmodulemd
+, libpeas
+, librepo
+, libsmartcols
+, libsolv
+, libxml2
+, rpm
+, sdbus-cpp
+, sqlite
+, systemd
+, toml11
+, zchunk
+}:
+
+
+stdenv.mkDerivation rec {
+  pname = "dnf5";
+  version = "5.0.15";
+
+  src = fetchFromGitHub {
+    owner = "rpm-software-management";
+    repo = pname;
```

```suggestion
    repo = "dnf5";
```

See: https://github.com/nix-community/nixpkgs-lint/issues/21